website/routes.py

```diff
@@ -705,6 +705,22 @@ def make_url_map(app):
             OsfWebRenderer('public/resetpassword.mako', render_mako_string, trust=False)
         ),
 
+        # reset password get - institution
+        Rule(
+            '/resetpassword-institution/<uid>/<token>/',
+            'get',
+            auth_views.reset_password_institution_get,
+            OsfWebRenderer('public/resetpassword_institution.mako', render_mako_string, trust=False)
+        ),
+
+        # reset password post - institution
+        Rule(
+            '/resetpassword-institution/<uid>/<token>/',
+            'post',
+            auth_views.reset_password_institution_post,
+            OsfWebRenderer('public/resetpassword_institution.mako', render_mako_string, trust=False)
+        ),
```

Comment:
As mentioned earlier, update both to use the existing view.

Reply:
Affected by the `process_rules` overwrite issue, too.